src/deserialize_sequence.cc

```diff
@@ -307,6 +314,11 @@ std::vector<Tag> parse_tags(gul14::string_view str)
     return tags;
 }
 
+bool parse_automation(gul14::string_view str)
+{
+    return gul14::trim(str) == "true" ? true : false;
```

Comment:
The other "parse" functions throw an exception if the format is not right, so that we can detect malformed input files. In comparison, this one is very forgiving and even parses an empty string into `false`. Maybe we should only allow "true" and "false".

Reply:
It was a first approach but I will change it to only accept `true` or `false` and throw otherwise an exception.

Reply:
Fixed. Now sets on the given string to true or false exactly.